src/content/docs/dropins/product-details/pdp-containers.mdx

```diff
@@ -141,6 +128,17 @@ Triggered when the Add to Cart button is clicked. Receives an object with `sku`,
 onAddToCart: (values) => void;
 ```
 
+## zoomType
+
+This property can take one of two string values: `zoom` or `overlay`. It is used to enhance product image viewing by providing zoom or overlay functionality:
+
+- `overlay` (_default_): Provides a larger, distraction-free view of a product image. The clicked image is opened in full screen.
+- `zoom`: Provides a close-up view of the product's image in the carousel. The clicked image is not opened in full screen.
+
+## closeButton
+
+This property is a boolean that controls the visibility of a close button on modals and overlays. It can only be used in combination with `zoomType`. The default value is `false`. If `true`, the close button is shown; if `false` or not provided, the close button is not shown.
```

Comment:
This can be removed ⬇️. It doesn't apply to modals or overlays, just to the zoom functionality.
> This property is a boolean that controls the visibility of a close button on modals and overlays.